src/Keboola/StorageApi/Client.php

```diff
@@ -160,14 +161,15 @@ public function __construct(array $config = [])
             $this->setJobPollRetryDelay(createSimpleJobPollDelay());
         }
 
-        $this->initClient();
+        $this->initClient($config);
         $this->tokens = new Tokens($this);
     }
 
-    private function initClient()
+    private function initClient(array $config)
     {
         $handlerStack = HandlerStack::create([
             'backoffMaxTries' => $this->backoffMaxTries,
+            'handler' => $config['handler'] ?? null,
```

Comment:
Bych přísahal, že tam tahle možnost vždycky byla

Reply:
No mame to v tech novejsich klientech, napr. internal API https://github.com/keboola/job-queue/blob/0a154e1339cb5fbee56dfd4ab2c9d300b31b95e1/libs/internal-api-php-client/src/Client.php#L352

Reply:
Který byly právě okopírovaný ze storage klienta :)